odl/solvers/util/callback.py

```diff
@@ -575,6 +575,69 @@ def __repr__(self):
         return '{}(seconds={})'.format(self.__class__.__name__, self.seconds)
 
 
+class CallbackShowConvergence(SolverCallback):
+
+    """Displays a convergence plot."""
+
+    def __init__(self, functional, title='convergence',
+                 logx=False, logy=False, **kwargs):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        functional : callable
+            Function that is called with the current iterate and returns the
+            function value.
+        title : str, optional
+            Title of the plot.
+        logx : bool, optional
+            If true, the x axis is logarithmic.
+        logx : bool, optional
+            If true, the y axis is logarithmic.
+        kwargs :
+            Additional parameters passed to the scatter function.
```

Comment:
"scatter plotting function" perhaps. Or even make a link to the matplotlib function.